messages/list/messages.go

```diff
@@ -0,0 +1,8 @@
+package list
+
+var (
+	Usage            = "list [flags]"
```

Comment:
```suggestion
	Usage            = "list [resource]"
```

Reply:
We need to use Flags there @gabriel-azion 
It's part of the Cobra framework nomenclature